frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -501,11 +501,20 @@ const PropertyListView: React.FC = () => {
                 const response: any = await callApi(apiProperty.id, apiProperty);
                 nextProperties = nextProperties.map((item, index: number) => {
                   if (index === change.rowId) {
-                    item = { ...item, ...flattenProperty(response) } as any;
+                    item = {
+                      ...item,
+                      ...flattenProperty(response),
+                      assessed: item.assessed,
+                      netBook: item.netBook,
+                      market: item.market,
+                    } as any;
                   }
                   return item;
                 });
               }
+
+              // Set key in localstorage to force the properties search to invalidate cached cache
+              localStorage.setItem(propertiesListViewChangedKey, 'true');
```

Comment:
is there no way to invalidate the cache directly? this seems a bit hacky.

Reply:
I could not find a better way, but I welcome better ideas

Reply:
Possibly you could include an extra query string parameter in the request. The cache i believe is based on that.  Then just make a request with a new value.  But you would need to test if the cache header will use a random query string

Reply:
Couple of ideas here: 
https://stackoverflow.com/questions/49263559/using-javascript-axios-fetch-can-you-disable-browser-cache